### ISSUE TITLE
sourcing  .vimrc  on mac

### ISSUE THREAD

**Issue by DavisGuohua-xie** [VundleVim/Vundle.vim#867]
$ vim -version
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Apr 24 2018 05:16:21)
Garbage after option argument: "-version"
More info with: "vim -h"

$ cat .vimrc 
set nocompatible              " be iMproved, required
filetype on
filetype off                  " required

" set the runtime path to include Vundle and initialize
set rtp+='~/.vim/bundle/Vundle.vim'

call vundle#begin()

" let Vundle manage Vundle, required
Plugin 'VundleVim/Vundle.vim'
call vundle#end()  

$ pwd
/Users/davisxie

$ pwd
/Users/davisxie/.vim/bundle/Vundle.vim
davisxie at davisxies-MacBook-Pro in ~/.vim/bundle/Vundle.vim on master*

$ source .vimrc 
-bash:  set the runtime path to include Vundle and initialize
set rtp+='~/.vim/bundle/Vundle.vim'

call vundle#begin()

: No such file or directory
-bash: Plugin: command not found
-bash: .vimrc: line 13: syntax error near unexpected token `('
-bash: .vimrc: line 13: `call vundle#end()  '

**Comment by jrebs** [comment]
You don't source .vimrc from the bash shell, you source it from within vim.

**Comment by davidlowryduda** [comment]
@DavisGuohua-xie have you now resolved your problem? Should this issue be closed?

**Comment by ctataryn** [comment]
@DavisGuohua-xie Vim will automatically re-source `.vimrc` when you restart it.  Otherwise, you may do this in the editor like so:

`:source $MYVIMRC`

**Comment by DavisGuohua-xie** [comment]
Yes. i "revolved" now. I just didn't source .vimrc. it works, so did i
really resolved it ??  if Vim automatically re-source .vimrc, why it never
get complaint with error? I will try :source $MYVIMRC when my mac come back
from Apple.  Thank you.
